migrations/20190723034059-add-type-to-expense-table.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn('Expenses', 'type', {
```

Comment:
Any column added to the `Expenses` table must be added to `ExpenseHistories` as well. Same goes for the followings:

> 'Collectives', 'Comments', 'Expenses', 'Subscriptions', 'Updates'

You were not supposed to know it, it's not documented. I'm adding a test right now to ensure that we keep the models in sync: https://github.com/opencollective/opencollective-api/pull/2323

In the mean time you can check https://github.com/opencollective/opencollective-api/blob/53ea1c189c83ae506e53155359ddb1c22e8c6079/migrations/20190327172521-add-is-pledged-to-collectives.js#L14 for an example of how we do that.